docs/source/routing_table_tools_doctest.rst

```diff
@@ -120,8 +120,8 @@ minimised.
     ... ]
 
     >>> # Minimise the routing table as much as possible
-    >>> from rig.routing_table import minimise
-    >>> minimised = minimise(original, target_length=None)
+    >>> from rig.routing_table import minimise_table
```

Comment:
The `minimise` prototype above presumably needs modifying to be called `minimise_table` to match this (sensible!) change.